chain-impl-mockchain/src/vote/committee.rs

```diff
@@ -22,7 +22,7 @@ impl CommitteeId {
     pub const COMMITTEE_ID_SIZE: usize = 32;
 
     /// returns the identifier encoded in hexadecimal string
-    pub fn to_hex(&self) -> String {
+    pub fn to_hex(self) -> String {
```

Comment:
doesn't it make sense to take by reference instead here?

Reply:
Kind of, but clippy complains about naming with `to_*` and the type being copy